zerver/actions/message_edit.py

```diff
@@ -336,6 +336,19 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
     send_event(user_profile.realm, event, list(map(user_info, ums)))
 
 
+def get_visibility_policy_after_merge(
+    orig_topic_visibility_policy: int, target_topic_visibility_policy: int
+) -> int:
```

Comment:
A comment here explaining the algorithm's intent of "Whichever of the two options is most visible is what we keep" with some of the motivation would be very helpful.

Reply:
Added a comment.